cos-fleetshard-operator-debezium/src/main/java/org/bf2/cos/fleetshard/operator/debezium/DebeziumOperandController.java

```diff
@@ -92,10 +116,11 @@ public DebeziumOperandController(KubernetesClient kubernetesClient, DebeziumOper
             .addToData(KAFKA_CLIENT_SECRET_KEY, serviceAccountSpec.getClientSecret())
             .build();
 
-        PodTemplate podTemplate = new PodTemplateBuilder()
-            .withImagePullSecrets(IMAGE_PULL_SECRET)
+        ConfigMap kafkaConnectMetricsConfigMap = new ConfigMapBuilder()
```

Comment:
This is fine for now but given that the configmap is always the same, can we create a config map once when the operator starts ? 

Eventually such config map can also be provided as part of the addon.

Reply:
See line 71 in the same file: https://github.com/bf2fc6cc711aee1a0c2a/cos-fleetshard/blob/3e9bcc813b4cbe0123b155020d34c98aca40c51b/cos-fleetshard-operator-debezium/src/main/java/org/bf2/cos/fleetshard/operator/debezium/DebeziumOperandController.java#L71
:wink:

Reply:
ah great, didn't spot it

Reply:
The intention is/was to maybe have some connector specific config eventually. When we see it won't be needed, I agree to put it into the bundle/addon config is the way to go. That's why I put the comment above.